src/helpers/remind-and-remove.ts

```diff
@@ -97,39 +99,54 @@ async function shouldDisplayRemainingExtensionsReminder(context: ContextPlugin,
   return lastRemainingExtensionsValue !== issueAndPrTargets.remainingExtensions;
 }
 
-async function buildReminderMessage(context: ContextPlugin, args: { remainingExtensions: number; extensionsLimit: number } & IssuePrTarget) {
-  return !context.config.negligenceThreshold ||
-    !context.config.availableDeadlineExtensions.enabled ||
-    !(await shouldDisplayRemainingExtensionsReminder(context, args))
+async function buildReminderMessage(context: ContextPlugin, args: { issue: ListIssueForRepo; extensions: Extensions } & IssuePrTarget) {
+  const logins = args.issue.assignees
+    ?.map((o) => o?.login)
+    .filter((o) => !!o)
+    .join(", @");
+  const shouldDisplayExtensions =
+    context.config.negligenceThreshold && context.config.availableDeadlineExtensions.enabled && (await shouldDisplayRemainingExtensionsReminder(context, args));
+  const currentExtensionsMilestone = args.extensions.extensionsLimit - args.remainingExtensions + 1;
+  const reminderContent = !shouldDisplayExtensions
     ? "this task has been idle for a while"
-    : `you have used <code>**${args.extensionsLimit - args.remainingExtensions + 1}**</code> of <code>**${args.extensionsLimit}**</code> available deadline extensions`;
+    : `you have used <code>**${currentExtensionsMilestone}**</code> of <code>**${args.extensions.extensionsLimit}**</code> available deadline extensions`;
+  const message = [`@${logins}, ${reminderContent}.`, "Please provide an update on your progress."];
+
+  if (shouldDisplayExtensions && args.extensions.assignmentDate) {
+    message.push(
+      `The new deadline is on ${new Date(
```

Comment:
Can you match the style 

https://github.com/Meniole/daemon-disqualifier/issues/34#issuecomment-2709976511